### DISCUSSION TITLE
I used the Chainlink VRF but it was failed, To use this, what we should set?

### DISCUSSION THREAD

**Discussion by kaizendev2015** [samurai-techlead/Coinflip-Contract-Simple-VRF#4] [Q&A]
![1](https://user-images.githubusercontent.com/94433565/178003273-e409c275-602c-4010-8752-1b1fc6d482d1.png)

**Comment by samurai-techlead** [comment]
Seems like you didn't deposit the Link token to the smart contract.
To use the Chainlink VRF, you should purchase the link token.
Thanks.
